filters/auth/oidc_introspection.go

```diff
@@ -131,7 +131,12 @@ func (filter *oidcIntrospectionFilter) Request(ctx filters.FilterContext) {
 		return
 	}
 
-	sub := token.Claims["sub"].(string)
+	sub, ok := token.Claims["sub"].(string)
+	if !ok {
+		unauthorized(ctx, sub, invalidSub, "", "")
```

Comment:
Here the sub will be empty? Maybe then pass empty value (`""`) or a sentinel value.

Reply:
I think we can also pass r.Host like in the invocations above.